include/unistd.h

```diff
@@ -308,6 +308,7 @@ extern "C"
 /* Task Control Interfaces */
 
 pid_t   fork(void);
+pid_t   up_fork(void);
```

Comment:
I think this is improper place for internal prototype.It is already available in `include/nuttx/arch.h`, so why do we need it here?